lib/utils/log-file.js

```diff
@@ -208,6 +207,7 @@ class LogFiles {
       )
 
       // Always ignore the currently written files
+      const { glob } = require('glob')
```

Comment:
This could be a problem under the situation where we somehow remove `glob` as a dependency from npm in a way that makes it hoisted.  This is currently pretty unlikely but we can't rule it out as impossible.

However, this entire block is in a try/catch that has a worst case scenario of logging a warning that the file couldn't be cleaned up, so the global npm install will still succeed.

Reply:
Should we add a test just to ensure the glob is still a dependency?

Reply:
I'm not sure how to address this long term.  I think for now a comment on the lazy load to make sure it stays in a try/catch should be good enough for now.

Reply:
I'm not convinced this is a problem since we have a test for this case. In smoke-tests we globally install "npm" except it's just an empty package with the same npm. Wouldn't that cover what we are worried about here?